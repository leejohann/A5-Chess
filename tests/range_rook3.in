move 0 6 0 5
move 0 1 0 3
move 0 5 0 4
move 0 0 0 2
quit
