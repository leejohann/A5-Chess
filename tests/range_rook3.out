RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
_ _ _ _ 
pppppppp
rnbqkbnr

it is White's turn
RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
p _ _ _ 
 ppppppp
rnbqkbnr

it is Black's turn
RNBQKBNR
_PPPPPPP
 _ _ _ _
P _ _ _ 
 _ _ _ _
p _ _ _ 
 ppppppp
rnbqkbnr

it is White's turn
RNBQKBNR
_PPPPPPP
 _ _ _ _
P _ _ _ 
p_ _ _ _
_ _ _ _ 
 ppppppp
rnbqkbnr

it is Black's turn
 NBQKBNR
_PPPPPPP
R_ _ _ _
P _ _ _ 
p_ _ _ _
_ _ _ _ 
 ppppppp
rnbqkbnr

it is White's turn

